lib/mix/test/mix/tasks/help_test.exs

```diff
@@ -29,6 +29,20 @@ defmodule Mix.Tasks.HelpTest do
     end
   end
 
+  test "help list alias task", context do
```

Comment:
Minor nitpick: I think it should be "lists", and maybe "help lists all aliases"?